source/guides/index.rst

```diff
@@ -55,4 +55,4 @@ Stacks
    :glob:
 
    stacks/*
-   dlrs2
+   stacks/dlrs/*

```

Comment:
Thanks for the correction. Working too fast to meet the deadline.
Previous PR showed passing, but apparently I didn't wait long enough for Travis to run the VM.

Reply:
No worries. :-) Happens to us all.